docs-ref-services/intune.md

```diff
@@ -24,17 +24,33 @@ Azure Intune is a cloud-based enterprise mobility management (EMM) service that
 
 Intune integrates closely with Azure Active Directory (Azure AD) for identity and access control, and Azure Information Protection for data protection.
 
-## Install the modules with npm
+## Management Package
 
-Use npm to install the Azure Intune modules for Node.js
+### Install npm module
+
+Use npm to install the Azure Intune module for Node.js
 
-### Management
 ```bash
 npm install azure-arm-intune
 ```
 
-##Example
+### Example
+
+Authenticate, create a client and retrieve a location by host name.
+
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const IntuneResourceManagementClient = require('azure-arm-intune');
+
+msRestAzure.interactiveLogin().then(credentials => {
+  const client = new IntuneResourceManagementClient(credentials);
+  client.getLocationByHostName(null).then(result => {

```

Comment:
That's a goooooood question. I _think_ this works with `null` in place of an options object (looking at the code), but I need to check with someone who knows. I don't have permissions to read over the Microsoft InTune scope, so I can't actually get that far.